File: eternal_return/src/main/java/org/EternalReturn/ERAnimal/animals/actors/Boar.java
package org.EternalReturn.ERAnimal.animals.actors;

import org.EternalReturn.ERAnimal.ERAJEntity;
import org.EternalReturn.ERAnimal.ERAnimal;
import org.EternalReturn.System.PluginInstance;
import org.bukkit.Location;
import org.bukkit.entity.Entity;
import org.jetbrains.annotations.NotNull;

public class Boar extends ERAnimal {
    public Boar(@NotNull ERAJEntity erAjEntity, @NotNull Location location) {
        super(erAjEntity, PluginInstance.getEREngine().createOrientedBox(location,1.5/2,1.5/2,2.0/2));
        this.ajEntity.registerAnimation("ready",2.0d);
        this.ajEntity.registerAnimation("attack",2.3d);
        this.ajEntity.registerAnimation("move",3.5d);
        this.ajEntity.registerAnimation("skill",3.25d);
        this.ajEntity.registerAnimation("death",2.0d);
    }
}

File: eternal_return/src/main/java/org/eternalreturn/eranimal/animals/actors/Boar.kt
package org.eternalreturn.eranimal.animals.actors

import org.bukkit.Location
import org.eternalreturn.eranimal.ERAJEntity
import org.eternalreturn.eranimal.ERAnimal
import org.eternalreturn.system.EREngine

class Boar(engine: EREngine, erAjEntity: ERAJEntity, location: Location) :
    ERAnimal(engine, erAjEntity, 1.0, 1.0, 2.0, 0.0, 0.0, 0.0) {
    init {
        this.aJEntity.registerAnimation("ready", 2.0)
        this.aJEntity.registerAnimation("attack", 2.3)
        this.aJEntity.registerAnimation("move", 3.5)
        this.aJEntity.registerAnimation("skill", 3.25)
        this.aJEntity.registerAnimation("death", 2.0)
    }
}